tests/Feature/BrandControllerTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use App\Models\Brand;
+use App\Models\User;
+
+it('user can add a new brand', function () {
+
+    $user = User::factory()->create();
+    $response = $this->actingAs($user);
+
+    $response = $this->from(route('brand.create'))->post(route('brand.store'), [
+        'name' => 'test',
+    ]);
+
+    $response->assertStatus(302);
+    $response->assertRedirect(route('brand.index'));
+    $brand = Brand::first();
+
+    $this->assertEquals($brand->name, 'test');
```

Comment:
just checked that brand table column: nam have 'test' value.


read docs:
https://laravel.com/docs/10.x/database-testing#available-assertions